README.md

```diff
@@ -1,31 +1,127 @@
-# hot-glue
+# cyanoacrylate
 
 > Toolkit for large-scale automated traffic analysis of mobile apps on Android and iOS.
 
-<!-- TODO: A longer introduction to the module. -->
+This toolkit was designed to run traffic analyses on lots of apps without much user interaction, to anlyse the tracking behavior of mobile apps. It supports running apps on Android and iOS, currently on physical devices as well as an emulator for android. It uses mitmproxy to capture the device traffic and [appstraction](https://github.com/tweaselORG/appstraction) to instrument the devices.  
 
-## Installation
+The current features include:
 
-You can install hot-glue using yarn or npm:
+- Checking if device traffic is altered by a DNS blocker
+- Starting and managing an android emulator
+- Collecting HTTP(S) traffic in HAR format (requires a setup CA on the device)
+- Installation of split APKs
+
+**Currently, this module does only support POSIX environments. It will not run on Windows out of the box.**
+
+## Installation  
+
+You can install cyanoacrylate using yarn or npm:
 
 ```sh
-yarn add hot-glue
-# or `npm i hot-glue`
+yarn add cyanoacrylate
+# or `npm i cyanoacrylate`
+```
+
+Furthermore, you need to install some dependencies on the host machine and prepare your target device/emulator to allow for traffic inspection.
+
+### Host dependencies
+
+Look at the [host dependencies for appstraction](https://github.com/tweaselORG/appstraction#host-dependencies-for-android) for your use cases and follow the instructions to install those.
+
+Also, install `mitmproxy` using pip:
```

Comment:
Well, it kinda depends on whether we want to call mitmproxy in our own startup script, which would allow us to put the the ipc plugin into a PyPI package. If we want that,we need mitmproxy available to import and if we want venvs we need our users to manage and create the venv themselves. If we don't want the startup script, `pipx` is the much more comfortable option.